external-authority-sources.md

```diff
@@ -0,0 +1,275 @@
+              {
+                "value": "https://orcid.org/0000-0002-4271-0436",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "person.familyName": [
+              {
+                "value": "Smith",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "person.givenName": [
+              {
+                "value": "Dean",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ]
+        },
+        "type": "externalSourceEntry",
+        "_links": {
+          "authority": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/authorities/authors/entryValues/d4b5ca88-9d6d-4a87-b905-fef0f8cae26c"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/orcid/entryValues/0000-0002-4271-0436"
+          }
+        }
+      },
+      {
+        "id": "0000-0003-3681-2038",
+        "display": "Smith, Charles",
+        "value": "Smith, Charles",
+        "metadata": {
+            "dc.identifier.orcid": [
+              {
+                "value": "0000-0003-3681-2038",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "dc.identifier.uri": [
+              {
+                "value": "https://orcid.org/0000-0003-3681-2038",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "person.familyName": [
+              {
+                "value": "Smith",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "person.givenName": [
+              {
+                "value": "Charles",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ]
+        },
+        "type": "externalSourceEntry",
+        "_links": {
+          "entity": {
+            "href": "https://dspace7-internal.atmire.com/server/api/core/item/6fd90bf5-b84f-47b3-aaec-a55bde3a2a5a"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/externalsources/orcid/entryValues/0000-0003-3681-2038"
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+### single entry
+**GET /api/integration/externalsources/<:authority-name>/entryValues/<:entry-id>**
```

Comment:
here <:source-name> makes more sense to me than <:authority-name>

Reply:
This is indeed better to avoid confusion. I've updated it accordingly